src/containerapp/azext_containerapp/_params.py

```diff
@@ -85,7 +85,8 @@ def load_arguments(self, _):
         c.argument('max_replicas', type=int, help="The maximum number of replicas.")
         c.argument('scale_rule_name', options_list=['--scale-rule-name', '--srn'], help="The name of the scale rule.")
         c.argument('scale_rule_type', options_list=['--scale-rule-type', '--srt'], help="The type of the scale rule. Default: http.")
-        c.argument('scale_rule_http_concurrency', type=int, options_list=['--scale-rule-http-concurrency', '--srhc', '--srtc', '--scale-rule-tcp-concurrency'], help="The maximum number of concurrent requests before scale out. Only supported for http and tcp scale rules.")
+        c.argument('scale_rule_http_concurrency', type=int, options_list=['--scale-rule-http-concurrency', '--srhc'], help="The maximum number of concurrent http requests before scale out.")
+        c.argument('scale_rule_tcp_concurrency', type=int, options_list=['--scale-rule-tcp-concurrency', '--srtc'], help="The maximum number of concurrent tcp requests before scale out.")
```

Comment:
The newly added parameter `scale_rule_tcp_concurrency` should also have logic similar to `scale_rule_http_concurrency` in the custom method